React/Views/RCTView.m

```diff
@@ -154,33 +156,58 @@ - (NSString *)accessibilityLabel
   return RCTRecursiveAccessibilityLabel(self);
 }
 
+-(void)setAccessibilityActions:(NSArray *)actions
+{
+  if (!actions || !actions.count) {
+    return;
```

Comment:
It doesn't look like the dictionaries get emptied out if someone tries to remove actions, e.g. `[view setAccessibilityActions:@[]]`